src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/logger.html.twig

```diff
@@ -224,7 +225,7 @@
                     <a class="btn-link text-small sf-toggle" data-toggle-selector="#{{ context_id }}" data-toggle-alt-content="Hide full context">Show full context</a>
 
                     <div id="{{ context_id }}" class="context">
-                        <pre>{{ context_dump }}</pre>
+                        {{ dump(log.context) }}
```

Comment:
Maybe you could use `profiler_dump` to keep consistency with other panels until we use `dump` everywhere (ref https://github.com/symfony/symfony/issues/18149)

// cc @javiereguiluz